gcn/notices/fermi/gbm/strict/alert.2.example.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "https://gcn.nasa.gov/schema/main/gcn/notices/fermi/gbm/alert.schema.json",
+  "additional_info": "*** ABOUT THIS EXAMPLE: This is an example of a Fermi GBM Alert GCN notice. Only stricly necessary fields are present. This 'about' note is not going to be part of streamed/real notices. This field may be missing or empty in case of no comments. An example of a typical notice comment follows. ***\n\nLightcurve and other quicklook files will not be created until about 15 minutes after the trigger.",
```

Comment:
What is different between these examples, please add some info about their difference rather than number them in their name.

Reply:
Agree and done. Thanks!